x/gov/simulation/genesis_test.go

```diff
@@ -40,7 +40,7 @@ func TestRandomizedGenState(t *testing.T) {
 	var govGenesis types.GenesisState
 	simState.Cdc.MustUnmarshalJSON(simState.GenState[types.ModuleName], &govGenesis)
 
-	dec1, _ := sdk.NewDecFromStr("0.466000000000000000")
+	dec1, _ := sdk.NewDecFromStr("0.457000000000000000")
```

Comment:
i'm not 100% sure hot this value was calculated but the old value was giving error when i added edpedited_quorum_threshold